clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -24,6 +26,16 @@
 (define-constant no-fee-x-err (err u75))
 (define-constant no-fee-y-err (err u76))
 
+(define-data-var swap-shutdown-activated bool false)
+
+(define-public (toggle-swap-shutdown)
```

Comment:
Should we ensure test coverage there? I was unable to find some tests using this feature.

Reply:
We have tests on that, see here: https://github.com/arkadiko-dao/arkadiko/pull/316/files#diff-0e4d987d6791a7d5251a79a65aec837f12eb70f300be849696300b2564f5d464R753